pkgs/development/tools/build-managers/knit/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+    pname = "knit";
```

Use only two spaces as indentation.
```suggestion
  pname = "knit";
```